csharp/src/Google.Protobuf.Benchmarks/Program.cs

```diff
@@ -41,8 +42,36 @@ class Program
         // if you start from "-f net461", it goes horribly wrong)
         public static void Main(string[] args)
         {
+#if !RUN_PROFILER
             BenchmarkSwitcher.FromAssembly(typeof(Program).Assembly).Run(args);
+#else
+            // Used to run benchmark files in regular code for profiling purposes
+            ProfileBenchmark();
+#endif
         }
+
+#if RUN_PROFILER
```

Comment:
I can remove it if you want. I find it useful to leave it here and enable it with a compiler flag, rather than re-add it each time I want to profile benchmarks.

Reply:
Got it, but it seems arbitrary that with profiler this only runs one of the available benchmarks `SerializationBenchmark.ParseFromReadOnlySequence`.
If you believe this change is worth having, let's add this in a separate PR where it's going to be easier to review and we can also give more thought to deciding which benchmarks are the best ones to run under profiler etc.
I'd say let's remove from this PR.